### DISCUSSION TITLE
v-radio-group styling

### DISCUSSION THREAD

**Discussion by faaktap** [vuetifyjs/vuetify#14073] [Q&A]
Am I missing something, or are there no out-of-the-box styling for vuetify radio buttons?
v-text-field has styling like ---  outline, dense, rounded, shaped 

However, if you style these fields, and you use radio buttons with them, the radio buttons look kind of `alone`.

I guess I need to adapt the styling for the radio buttons on my own?
Or am I missing something?
Maybe radio buttons are going out of style?

![vuetifyRadioButtons](https://user-images.githubusercontent.com/8826110/130302768-8fc7d07a-cc34-46aa-9d7c-a266c6ba7512.jpg)

**Comment by jacekkarczmarczyk** [comment]
Yes, you'd need to style it by yourself, current design is in line with Material Design specs https://material.io/components/radio-buttons
